decoder/semantic_tokens_expr_test.go

```diff
@@ -1433,6 +1435,8 @@ EOT
 		},
 	}
 
+	ctx := context.Background()
```

Comment:
Although it won't make a difference in this particular test, it's typically best to start each (sub)test with a fresh/isolated context, i.e. move this under `t.Run()` below.